fluentforms/core/src/main/java/com/_4point/aem/fluentforms/api/assembler/AssemblerResult.java

```diff
@@ -0,0 +1,12 @@
+package com._4point.aem.fluentforms.api.assembler;
+
+import java.util.Map;
+
+import com._4point.aem.fluentforms.api.Document;
+
+public interface AssemblerResult {
+	
+	 Map<String,Document> getDocuments();
+	
```

Comment:
Again, only one attribute is implemented here when the Adobe class has [8 additional attributes](https://helpx.adobe.com/experience-manager/6-5/forms/javadocs/com/adobe/fd/assembler/client/AssemblerResult.html) in their result class.